src/Apps/Home/Components/HomeAuctionLotsForYouRail.tsx

```diff
@@ -0,0 +1,140 @@
+import { Shelf, Skeleton } from "@artsy/palette"
+import * as React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useSystemContext } from "System/useSystemContext"
+import { useTracking } from "react-tracking"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import {
+  ShelfArtworkFragmentContainer,
+  ShelfArtworkPlaceholder,
+} from "Components/Artwork/ShelfArtwork"
+import {
+  ActionType,
+  ClickedArtworkGroup,
+  ContextModule,
+  OwnerType,
+} from "@artsy/cohesion"
+
+import { HomeAuctionLotsForYouRail_artworksForUser$data } from "__generated__/HomeAuctionLotsForYouRail_artworksForUser.graphql"
+import { HomeAuctionLotsForYouRailQuery } from "__generated__/HomeAuctionLotsForYouRailQuery.graphql"
+import { extractNodes } from "Utils/extractNodes"
+
+interface HomeAuctionLotsForYouRailProps {
+  artworksForUser: HomeAuctionLotsForYouRail_artworksForUser$data
+}
+
+const HomeAuctionLotsForYouRail: React.FC<HomeAuctionLotsForYouRailProps> = ({
+  artworksForUser,
+}) => {
+  const { trackEvent } = useTracking()
+
+  const artworks = extractNodes(artworksForUser)
+
+  if (!artworks || artworks?.length === 0) {
+    return null
+  }
+
+  return (
+    <Shelf>
+      {artworks.map((artwork, index) => {
+        if (!artwork) {
+          return <></>
+        }
+
+        return (
+          <ShelfArtworkFragmentContainer
+            artwork={artwork}
+            key={index}
+            contextModule={ContextModule.auctionLots}
+            lazyLoad
+            onClick={() => {
+              //TODO: add tracking for Auction Lots For You
+              const trackingEvent: ClickedArtworkGroup = {
+                action: ActionType.clickedArtworkGroup,
+                context_module: ContextModule.auctionLots,
+                context_page_owner_type: OwnerType.home,
+                destination_page_owner_id: artwork.internalID,
+                destination_page_owner_slug: artwork.slug,
+                destination_page_owner_type: OwnerType.artwork,
+                type: "thumbnail",
+              }
+              trackEvent(trackingEvent)
+            }}
+          />
+        )
+      })}
+    </Shelf>
+  )
+}
+
+const PLACEHOLDER = (
+  <Skeleton>
+    <Shelf>
+      {[...new Array(8)].map((_, i) => {
+        return <ShelfArtworkPlaceholder key={i} index={i} />
+      })}
+    </Shelf>
+  </Skeleton>
+)
+
+export const HomeAuctionLotsForYouRailFragmentContainer = createFragmentContainer(
```

Comment:
Not a blocker - when possible, new components shouldn't use fragment container, but use the `useFragment` hook instead - https://artsy.slack.com/archives/C2TQ4PT8R/p1710950348303909

Reply:
Ah right, thanks for the reminder!